sources/Outputs/include/Dyd.h

```diff
@@ -29,6 +29,31 @@
 #include <unordered_map>
 #include <vector>
 
+namespace std {
+/**
+ * @brief specialization hash for generators model types
+ *
+ * For old compilers, hashs are not implemented for unordered_map when key is an enum class.
+ * see https://en.cppreference.com/w/cpp/utility/hash for template definition
+ */
+template<>
+struct hash<dfl::algo::GeneratorDefinition::ModelType> {
+  /// @brief Constructor
+  hash() {}
+  /**
+   * @brief Action operator
```

Comment:
Is it possible to avoid duplicating this code?

Reply:
I tried but there is a conflicts between the template we declare and the template implemented in libstd, the one that doesn't exist in the distante machine. I'm open to suggestions